.gitignore

```diff
@@ -8,6 +8,7 @@
 
 # dependencies
 /menelaos-front/node_modules
+/node_modules
```

Comment:
see comment on package-lock.json in project's root directory

Reply:
fixed